### PR TITLE
Lesson 15

### PR DESCRIPTION
**Pull request by vtx123**

### CODE REVIEW

File: Lesson15/Lesson15.cpp [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+{
+    std::string name;
+    unsigned age = 0;
+    unsigned markForMath = 60;
+    unsigned markForProg = 60;
+    unsigned markForBio = 60;
+    float averageEvaluation = 0.f;
+};
+
+float averageEvaluation(Student *st) {
```
Comment (bohdanpc) [inline Lesson15/Lesson15.cpp]: const Student&

File: Lesson15/Lesson15.cpp [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+    for (int i = 0; i < size; i++) {
+        if (st[i].averageEvaluation > maxEvaluation) {
+            maxEvaluation = st[i].averageEvaluation;
+            bestStudent = &st[i];
+        }
+    }
+    return bestStudent;
+}
+
+int coutSuccessedStudents(Student* st, const int size) {
```
Comment (bohdanpc) [inline Lesson15/Lesson15.cpp]: countSucceeded

File: Lesson15/Lesson15.cpp (hunk 1 above)
Comment (vtx123) [reply Lesson15/Lesson15.cpp]: fixed

File: Lesson15/Lesson15.cpp (hunk 2 above)
Comment (vtx123) [reply Lesson15/Lesson15.cpp]: fixed

**Pull request merged**